packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -166,6 +167,7 @@ export function PrivateBlockToolbar( {
 				{ ! isMultiToolbar &&
 					isLargeViewport &&
 					isDefaultEditingMode && <BlockParentSelector /> }
+				<BlockBindingsIndicator clientId={ blockClientId } />
```

Comment:
I believe we could move the conditional here as the rest of the component. See my other comment: [link](https://github.com/WordPress/gutenberg/pull/59185/files#r1505557629).

Reply:
That's a very good idea. It would allow us to get rid of the `useSelect` call in the child component.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/59185/commits/ec3885dcca2e3c851f6b1dd3a2461a8f8e300c91